core/services/keeper/orm.go

```diff
@@ -3,13 +3,24 @@ package keeper
 import (
 	"context"
 
-	"github.com/smartcontractkit/chainlink/core/services/bulletprooftxmanager"
-	"github.com/smartcontractkit/chainlink/core/services/keystore/keys/ethkey"
-
 	"gorm.io/gorm"
 	"gorm.io/gorm/clause"
+
+	"github.com/smartcontractkit/chainlink/core/services/bulletprooftxmanager"
+	"github.com/smartcontractkit/chainlink/core/services/keystore/keys/ethkey"
+	"github.com/smartcontractkit/chainlink/core/services/postgres"
 )
 
+// ORM implements ORM layer using PostgreSQL
+// TODO: Create an ORM interface and separate postgres implementation from this interface
```

Comment:
We generally try to avoid TODOs in code, especially when they might result in some discussion like this one.

Reply:
yeah in general I think we just leave it concrete until we do actually need to mock it for some reason. Sometimes testing with the db alone is enough

Reply:
Removed that TODO